docs/managing-policy.md

```diff
@@ -31,7 +31,7 @@ and blocking the build are all configured on a per-policy basis.
 1. Visit [Dashboard > Integrations](https://semgrep.dev/manage/notifications) to configure the services and name each of your integration channels. See [Integrations](integrations.md) for detailed instructions.
 2. From [Dashboard > Policies](https://semgrep.dev/manage/policy), select the policy you’d like to configure and add one or more integration channels from the Integrations drop-down menu.
 
-You can also check or uncheck the boxes to post pull request comments ([which requires a GitHub Token](integrations.md#automatic-pr-comments)) or to block the build on findings. Don't forget to click Save when you are finished editing!
+You can also check or uncheck the boxes to post pull request comments or to block the build on findings. Don't forget to click Save when you are finished editing!
```

Comment:
I would keep the link here to integrations.md#automatic-pr-comments since it does still contain info about GitHub only (not GitLab) and examples of what the comments look like
```suggestion
You can also check or uncheck the boxes to post [pull request comments](integrations.md#automatic-pr-comments) or to block the build on findings. Don't forget to click Save when you are finished editing!
```